### ISSUE TITLE
Automatic check (cronjob)

### ISSUE THREAD

**Issue by h1rule** [TSSSaver/TSSSaver#7]
Multiple ECID#s not checking. Its only checkin the first ECID.

It shows: "Total ECID 3" but only saves/checks does the last added ECID! Skipping the other 2

**Comment by 1Conan** [comment]
Well it works for me.

Mind giving me the console log?

**Comment by h1rule** [comment]
```
root@service:/var/www# cd /var/www/shsh2 && php cron.php
Total ECIDs: 2
Current ECID: 407267xxxxxx99289
Running: ./tsschecker -d 'iPhone9,4' -e '407267xxxxxx99289' -i '10.2.1' --buildid '14D27' -s --save-path shsh/407267xxxxxx99289/10.2.1/noapnonce
Running: ./tsschecker -d 'iPhone9,4' -e '407267xxxxxx99289' -i '10.2.1' --buildid '14D27' -s --apnonce 603be133ff0bdfa0f83f21e74191cf6770ea43bb --save-path shsh/407267xxxxxx99289/10.2.1/apnonce-603be133ff0bdfa0f83f21e74191cf6770ea43bb
Running: ./tsschecker -d 'iPhone9,4' -e '407267xxxxxx99289' -i '10.2.1' --buildid '14D27' -s --apnonce 352dfad1713834f4f94c5ff3c3e5e99477347b95 --save-path shsh/407267xxxxxx99289/10.2.1/apnonce-352dfad1713834f4f94c5ff3c3e5e99477347b95
Running: ./tsschecker -d 'iPhone9,4' -e '407267xxxxxx99289' -i '10.2.1' --buildid '14D27' -s --apnonce 42c88f5a7b75bc944c288a7215391dc9c73b6e9f --save-path shsh/407267xxxxxx99289/10.2.1/apnonce-42c88f5a7b75bc944c288a7215391dc9c73b6e9f
Running: ./tsschecker -d 'iPhone9,4' -e '407267xxxxxx99289' -i '10.2.1' --buildid '14D27' -s --apnonce 0dc448240696866b0cc1b2ac3eca4ce22af11cb3 --save-path shsh/407267xxxxxx99289/10.2.1/apnonce-0dc448240696866b0cc1b2ac3eca4ce22af11cb3
Running: ./tsschecker -d 'iPhone9,4' -e '407267xxxxxx99289' -i '10.2.1' --buildid '14D27' -s --apnonce 9804d99e85bbafd4bb1135a1044773b4df9f1ba3 --save-path shsh/407267xxxxxx99289/10.2.1/apnonce-9804d99e85bbafd4bb1135a1044773b4df9f1ba3
root@service:/var/www/shsh2# 
```

**Comment by 1Conan** [comment]
Try changing https://github.com/1Conan/TSSSaver/blob/master/cron.php#L37
To

```php
for($i = 0; $i <= $countSavedDevices; $i++) {
```

**Comment by h1rule** [comment]
Nothing changed :-/ still saves only the latest one ECID

**Comment by 1Conan** [comment]
could you email me a var_dump of your ECID db?

**Comment by h1rule** [comment]
sure, if you tell me how and whats your email?

**Comment by 1Conan** [comment]
Add this on cron.php#L34

```php
var_dump($savedDevices);
```

Email the whole output to admin[at]1conan.com

**Comment by h1rule** [comment]
done

**Comment by 1Conan** [comment]
Fixed in 78fd2c3

Forgot to add the changes here when I fixed that in tsssaver.1conaan.com

(Also, updating this repo eta: son)
